plugins/lookup/passwordstore.py

```diff
@@ -77,54 +79,88 @@
           - warn
           - empty
           - create
+      lock:
+        description:
+          - How to synchronize operations.
+          - The default of C(write) only synchs write operations.
+          - C(readwrite) synchs all operations (including read). This makes sure that gpg-agent is never called in parallel.
+          - C(none) doesn't do any synchronization.
+        ini:
+          - section: passwordstore_lookup
+            key: lock
+        type: str
+        default: write
+        choices:
+          - readwrite
+          - write
+          - none
+        version_added: 4.5.0
+      locktimeout:
+        description:
+          - Lock timeout in seconds.
```

Comment:
```suggestion
          - Lock timeout in seconds.
          - The default is 15 minutes.
```

Reply:
Specifying what 900 seconds means in minutes is mostly for convenience, since there's a significant amount of people who cannot do this in their head. I don't insist on this though, it's just a suggestion :)

Reply:
@felixfontein To me converting seconds is second - we couldn't afford parsing strings back in the 90s :)

Changed it now so that it requires a suffix to specify the unit (decided against a default, as this is also a matter of taste and potential confusion when reading configs).